src/components/pages/Cases/CaseTable.js

```diff
@@ -555,20 +563,18 @@ export default function CaseTable(props) {
    * getColumnSearchProps (above) was built with ant design, and is the header row of the table - it does not include the Filters div above it
    * the handleReset function is used in there to wipe the search of filters for specific columns, but I could not make it work for the search terms themselves
    */
-  const removeSearchTerm = filter => {
-    const term = filter.split(':')[0];
-    const key = filter.split(': ')[1];
-    const index = initialFilters[term].indexOf(key);
-    if (index > -1) {
-      initialFilters[term].splice(index, 1);
-    }
-    if (initialFilters[term].length === 0) {
-      initialFilters[term] = null;
-    }
-    setInitialFilters(initialFilters);
-    console.log(initialFilters);
+
+  const removeSearchTerm = (filterState, keyWord) => {
+    filterState[keyWord] = null;
+
+    setInitialFilters(filterState);
+
+    document.getElementById(keyWord).click();
+
+    return;
   };
 
```

Comment:
I might say the spacing for this function is different/not standardized with others throughout this file. You may want to remove the extra lines of spacing in 569, 571, and 573. Consistency is useful, but entirely a nitpick.